portal/server/controllers/dashboard/facilities/selected-filters.js

```diff
@@ -78,12 +38,12 @@ const selectedFilters = (submittedFilters) => {
     selected.push(obj);
   }
 
-  if (submittedFilters[CONSTANTS.FIELD_NAMES.FACILITY.HAS_BEEN_ISSUED]) {
+  if (submittedFilters.stage) {
```

Comment:
Should this not reference the constants file for field names (I don't really think this adds much to this file and happy to avoid this request if there's logic to this)

Reply:
I think I did this before I'd made that constant - as you say Im not the keenest on it but I'll add it back to preserve the pattern

Reply:
Thanks!